### ISSUE TITLE
[Feature] brcb原语没有加入

### ISSUE THREAD

**Issue by liuwenda4** [tile-ai/tilelang-ascend#1187]
## Feature Description
A clear and concise description of the feature you'd like to request.
[https://www.hiascend.com/document/detail/zh/CANNCommunityEdition/900beta2/API/ascendcopapi/atlasascendc_api_07_0089.html，这个原语](https://www.hiascend.com/document/detail/zh/CANNCommunityEdition/900beta2/API/ascendcopapi/atlasascendc_api_07_0089.html%EF%BC%8C%E8%BF%99%E4%B8%AA%E5%8E%9F%E8%AF%AD)
## Use Case
Describe the use case or scenario where this feature would be helpful.
xattention
## Proposed Solution
If you have a proposed solution or implementation idea, describe it here.

## Feature Type
- [ ] New operator/kernel (e.g., new GEMM variant, new attention mechanism)
- [x] New API/primitive (e.g., new T.xxx operation)
- [ ] Compiler optimization (e.g., new pass, performance improvement)
- [ ] New backend support (e.g., new hardware target)
- [ ] Developer tooling (e.g., debugging, profiling)
- [ ] Documentation improvement
- [ ] Other: [specify]

## Operator Details (if applicable)

**Operator Type:**
[e.g. GEMM, Flash Attention, Softmax, LayerNorm, Convolution, etc.]
Flash Attention
**Input/Output Specification:**
```
Input shapes and dtypes:
- A: [M, K], float16
- B: [K, N], float16

Output shapes and dtypes:
- C: [M, N], float16
```

**Performance Requirements:**
[e.g. target throughput, latency, memory bandwidth]

**Reference Implementation (if any):**
[e.g. PyTorch implementation, CUDA kernel, paper reference]

## Alternative Solutions
Describe any alternative solutions or features you've considered.

## Implementation Constraints (if known)

**Programming Mode Preference:**
- [ ] Developer mode (automatic)
- [x] Expert mode (manual control)
- [ ] Both modes supported

**Memory Constraints:**
[e.g. L1 buffer size, UB buffer size constraints]

**Target Hardware:**
- [x] Ascend A2
- [x] Ascend A3
- [ ] Other Ascend variants

## Additional Context
Add any other context, references, or screenshots about the feature request here.

## Willingness to Contribute
- [x] I'm willing to submit a PR for this feature
- [ ] I can help with testing
- [ ] I can provide reference implementations
- [ ] I need guidance to get started
